### PR TITLE
[Integrations][Gitlab] Align default resources with the examples in Port Docs

### PR DESCRIPTION
**Pull request by Tankilevitch**
# Description

What -
Why -
How -

## Type of change

Please leave one option from the following and delete the rest:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Non-breaking change (fix of existing functionality that will not change current behavior)
- [ ] Documentation (added/updated documentation)

### CODE REVIEW

File: integrations/gitlab/.port/resources/port-app-config.yaml [hunk 1]
```diff
@@ -3,21 +3,24 @@ deleteDependentEntities: true
...
           properties:
-            url: .web_link
+            url: .web_url
+            readme: file://README.md
             description: .description
+            language: .__languages | to_entries | max_by(.value) | .key
             namespace: .namespace.name
-            full_path: .namespace.full_path
+            fullPath: .namespace.full_path
+            defaultBranch: .default_branch
```
[the remark this answers is not in the document]
Comment (Tankilevitch) [reply integrations/gitlab/.port/resources/port-app-config.yaml:20]: yes :)

File: integrations/gitlab/.port/resources/blueprints.json [hunk 2]
```diff
@@ -1,29 +1,46 @@
...
     "schema": {
       "properties": {
         "url": {
           "title": "URL",
-          "format": "url",
-          "type": "string"
+          "type": "string",
+          "format": "url"
+        },
+        "readme": {
```
[the remark this answers is not in the document]
Comment (Tankilevitch) [reply integrations/gitlab/.port/resources/blueprints.json:13]: I guess we didn't 😅

File: integrations/gitlab/.port/resources/port-app-config.yaml [hunk 3]
```diff
@@ -50,7 +53,6 @@ resources:
             updatedAt: .updated_at
             description: .description
             link: .web_url
-            labels: '[.labels[]]'
-            object: .
```
[the remark this answers is not in the document]
Comment (Tankilevitch) [reply integrations/gitlab/.port/resources/port-app-config.yaml:54]: I think this was a left over from some test session I guess. using `.` will paste all the dictionary response.

**Review by MPTG94 — APPROVED** [review]
LGTM

**Pull request merged**